gc/base/TLHAllocationSupport.hpp

```diff
@@ -83,14 +83,25 @@ class MM_TLHAllocationSupport
 
 	const bool _zeroTLH; /**< if true this TLH is primary (might be cleared by batchClearTLH), if false this is secondary TLH (and it would not be cleared ever) */
 
+	uintptr_t _reservedBytes; /**< Number of bytes reserved in the TLH by collector. If set, we are guaranteed to have this remaining size available when we flush/clear TLH. */
 public:
+	MMINLINE bool containsObjs() {
+		return ((NULL != getBase()) && (NULL != getTop()) && (getUsedSize() > 0));
```

Comment:
wouldn't just _getUsedSize() > 0_ suffice?

Reply:
Yes that should do